cypress/integration/Manual Listing Pages/ManualListingPages.feature

```diff
@@ -0,0 +1,157 @@
+            | desktop           | /about-cancer/treatment/clinical-trials/testicular-cancer | Clinical Trials to Treat Testicular Cancer |
+            | mobile            | /about-cancer/treatment/clinical-trials/testicular-cancer | Clinical Trials to Treat Testicular Cancer |
+            | desktop           | /about-cancer/treatment/clinical-trials/thymoma-thymic-carcinoma | Clinical Trials to Treat Thymoma and Thymic Carcinoma |
+            | mobile            | /about-cancer/treatment/clinical-trials/thymoma-thymic-carcinoma | Clinical Trials to Treat Thymoma and Thymic Carcinoma |
+
+    Scenario Outline: Manual trial listing page with intro text
+        Given screen breakpoint is set to "<screen breakpoint>"
+        When user is navigating to "/about-cancer/treatment/clinical-trials/covid-19"
+        Then page title is "NCI-Supported Clinical Trials for Coronavirus"
+        And intro text 1 paragraph is "Clinical trials are research studies that involve people. Through clinical trials, doctors find new ways to improve treatments and the quality of life for people with disease."
+        And intro text 2 paragraph is "NCI supports clinical trials to test promising treatments for patients with coronavirus disease 2019 (COVID-19) and to learn more about the disease's effects on the body. Some of the trials are specifically for patients with cancer. Your doctor can help you decide if a trial is right for you."
+        And "Clinical trials" link has href "/publications/dictionaries/cancer-terms/def/clinical-trial"
+        And "quality of life" link has href "/publications/dictionaries/cancer-terms/def/quality-of-life"
+
+        Examples:
+            | screen breakpoint |
+            | desktop           |
+            | mobile            |
+
+
+    Scenario Outline: Pager functionality
+        Given screen breakpoint is set to "<screen breakpoint>"
+        When user is navigating to "<path>"
+        And  the page displays pager info "Trials 1-N of N"
+        And the 1 pager is displayed with the following "<pager items>"
+        And the page "1" is highlighted
+        When user clicks on "Next >" button
+        Then the url has path "<path>" with query "?pn=2"
+        And the page "2" is highlighted
+        When the user navigates to "<path>" with non-existent page "pn=900"
+        Then the text "There are currently no available trials." appears
+
+        Examples:
+            | screen breakpoint | path                                                   | pager items  |
+            | desktop           | /about-cancer/treatment/clinical-trials/cam-procedures | 1,2,3,Next > |
+            | desktop           | /about-cancer/treatment/clinical-trials/covid-19       | 1,2,Next > |
+            | desktop            | /about-cancer/treatment/clinical-trials/adult-metastatic-brain-tumors | 1,2,3,4,5,Next > |
```

Comment:
you want to test across different breakpoints  - please dilute this column  with  tablet and mobile for some urls